komga-webui/src/locales/en.json

```diff
@@ -335,7 +335,7 @@
       "field_link_url_error_protocol": "Must be http or https",
       "field_link_url_error_url": "Must be a valid URL",
       "field_number": "Number",
-      "field_number_sort": "Sort Number",
```

Comment:
This is very difficult to translate, but easy to get wrong.
Ending up with "Sort (the) number" as if for a button is likely.

Reply:
> Ending up with "Sort (the) number" as if for a button is likely.

you assume that the translator doesn't understand the context of what he is translating. One should not translate an app without having used it and knowing what it means, no?

Reply:
You assume having used the app means having seen every string, including all errors.
The Spanish translation is wrong, and there are more translations adopting title case here. https://hosted.weblate.org/translate/komga/webui/nb_NO/?checksum=c692bebaf4eb1cc3

Reply:
🤷🏻